src/stream/src/executor/source/source_executor.rs

```diff
@@ -235,9 +235,10 @@ impl<S: StateStore> SourceExecutor<S> {
     ) -> StreamExecutorResult<()> {
         let core = self.stream_source_core.as_mut().unwrap();
         tracing::warn!(
-            "stream source reader error, actor: {:?}, source: {:?}",
-            self.actor_ctx.id,
-            core.source_id,
+            error = %e.as_report(),
```

Comment:
On the sink side, an error would trigger actor exit and recovery loop, with the error `Debug`ged with backtrace. Shall we follow the same practice here for source and switch from `Display` to `Debug`?
https://github.com/risingwavelabs/risingwave/blob/71898b4873a32b9df911a8157864100c0dfab27d/src/stream/src/task/stream_manager.rs#L694-L695